content/id/docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -0,0 +1,72 @@
+ - Apakah kamu **hanya ingin mencoba kluster Kubernetes**, atau kamu ingin ikut aktif melakukan **pengembangan kode dari proyek Kubernetes**? Jika jawabannya yang terakhir, pilihlah distro yang aktif dikembangkan. Beberapa distro hanya menggunakan rilis *binary*, namun menawarkan lebih banyak variasi pilihan.
+ - Pastikan kamu paham dan terbiasa dengan beberapa [komponen](/docs/admin/cluster-components/) yang dibutuhkan untuk menjalankan sebuah kluster.
+
+Catatan: Tidak semua distro aktif dikelola. Pilihlah distro yang telah diuji dengan versi terkini dari Kubernetes.
+
+## Mengelola Kluster
+
+* [Mengelola kluster](/docs/tasks/administer-cluster/cluster-management/) akan menjabarkan beberapa topik terkait *lifecycle* dari kluster: membuat kluster baru, melakukan *upgrade* pada *node master* dan *worker*, melakukan pemeliharaan *node* (contoh: *upgrade* kernel), dan melakukan *upgrade* versi Kubernetes API pada kluster yang sedang berjalan.
+
+* Pelajari bagaimana cara [mengatur *node*](/docs/concepts/nodes/node/).
+
+* Pelajari bagaimana cara membuat dan mengatur [kuota *resource*](/docs/concepts/policy/resource-quotas/) untuk *shared* kluster.
+
+## Mengamankan Kluster
+
+* [Sertifikat (*certificate*)](/docs/concepts/cluster-administration/certificates/) akan menjabarkan langkah-langkah untuk membuat sertifikat menggunakan beberapa *tool chains*.
+
+* [Kubernetes *Container Environment*](/docs/concepts/containers/container-environment-variables/) akan menjelaskan *environment* untuk kontainer yang dikelola oleh Kubelet pada Kubernetes *node*.
+
+* [Mengontrol Akses ke Kubernetes API](/docs/reference/access-authn-authz/controlling-access/) akan menjabarkan bagaimana cara mengatur *permission* untuk pengguna dan *service account*.
+
+* [Autentikasi](/docs/reference/access-authn-authz/authentication/) akan menjelaskan autentikasi di Kubernetes, termasuk ragam pilihan autentikasi.
+
+* [Otorisasi](/docs/reference/access-authn-authz/authorization/) dibedakan dari autentikasi, digunakan untuk mengontrol bagaimana *HTTP call* ditangani.
+
+* [Menggunakan *Admission Controllers*](/docs/reference/access-authn-authz/admission-controllers/) akan menjelaskan *plug-in* yang akan mengintersep permintaan ke server Kubernetes API setelah autentikasi dan otorisasi dilakukan.
+
+* [Menggunakan Sysctls pada Kluster Kubernetes](/docs/concepts/cluster-administration/sysctl-cluster/) akan menjabarkan tentang cara menggunakan perintah `sysctl` pada *command-line* untuk mengatur parameter kernel.
+
+* [Audit](/docs/tasks/debug-application-cluster/audit/) akan menjelaskan bagaimana cara berinteraksi dengan log audit Kubernetes.
+
+### Mengamankan Kubelet
+  * [Komunikasi Master-Node](/docs/concepts/architecture/master-node-communication/)
+  * [TLS *bootstrapping*](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/)
+  * [Autentikasi/Otorisasi Kubelet](/docs/admin/kubelet-authentication-authorization/)
+
+## Opsi *Cluster Service*
```

Comment:
use Kluster to be consistent with the rest?

Reply:
I rewording those to be Layanan Tambahan Kluster. Sounds good?

Reply:
sounds good :+1: